website/content/en/preview/upgrade-guide.md

```diff
@@ -102,6 +102,9 @@ Snapshot releases are tagged with the git commit hash prefixed by the Karpenter
 
 ## Released Upgrade Notes
 
+### Upgrading to v0.29.0+
```

Comment:
Can you add the relevant breaking change notice for `v0.28.0`? We should have a note that the default port for the pod changed in `v0.28.0` so that users that are scraping pods on that version are aware that this changed.